src/commands/sourcemaps/inject.rs

```diff
@@ -0,0 +1,150 @@
+use std::fs::File;
+use std::io::{BufRead, BufReader, Seek, Write};
+use std::path::{Path, PathBuf};
+
+use anyhow::{bail, Context, Result};
+use clap::{Arg, ArgMatches, Command};
+use glob::glob;
+use log::{debug, warn};
+use serde_json::Value;
+use uuid::Uuid;
+
+const CODE_SNIPPET_TEMPLATE: &str = r#"!function(){try{var e="undefined"!=typeof window?window:"undefined"!=typeof global?global:"undefined"!=typeof self?self:{},n=(new Error).stack;n&&(e._sentryDebugIds=e._sentryDebugIds||{},e._sentryDebugIds[n]="__SENTRY_DEBUG_ID__")}catch(e){}}()"#;
+const DEBUGID_PLACEHOLDER: &str = "__SENTRY_DEBUG_ID__";
+
+pub fn make_command(command: Command) -> Command {
+    command
+        .about("Fixes up JavaScript source files and sourcemaps with debug ids.")
+        // TODO: What are these {n}{n}s? They show up verbatim in the help output.
```

Comment:
They suppose to be terminal line-break specific to clap, but for some reason you say they stopped working? 🤔 

https://github.com/clap-rs/clap/blob/ad5d67623a89415c7f5f7e1f7a47bf5abdfd0b6c/src/output/help_template.rs#L1018

Reply:
The placeholders have apparently been in the `send_envelope-help` test case since the beginning.

https://github.com/getsentry/sentry-cli/blame/master/tests/integration/_cases/send_envelope/send_envelope-help.trycmd